config/locales/en.yml

```diff
@@ -2322,12 +2323,11 @@ en:
   hide_desc: |
     Hiding content will remove it from public view, but people will still be
     able to see that something was hidden, and site curators and the author of
-    the content will still be able to see the hidden content. Generally, this
-    only applies to written content, e.g. if you hide the content of an
-    identification, the identification is still visible but the identifier's
-    remarks are hidden. You cannot undo this, and only site staff can unhide
-    content once it has been hidden, so be careful and only use this for very
-    objectionable content.
+    the content will still be able to see the hidden content. Hidden
+    identifications will not contribute to an observation's Community Taxon.
```

Comment:
FWIW, I think it's better to change the key when meaning of the content changes. That way translators get notified about new strings to translate. I don't think they get notified in situations like this. The approval just gets removed.